tlscanary/template/js/report_page.js

```diff
@@ -27,19 +27,18 @@ function makeChartTab(uriList, fieldName) {
 
 function makeFieldControl(fieldName) {
   var html = "";
-  html += "<h3>Field:&nbsp;&nbsp;<select id=\"fieldNames\" name=\"fieldNames\" >";
+  html += "<h3>Field:&nbsp;&nbsp;<select id='fieldNames' name='fieldNames' >";
   var columns = getVisibleColumns();
-  for (var i = 0;i < columns.length;i++)
-  {
-    html += "<option value=\"" + columns[i] + "\"";
-    if ( columns[i] === fieldName )
+  for (var i = 0;i < columns.length;i++) {
+    html += "<option value='" + columns[i] + "'";
```

Comment:
More hardquote. I won't mark them all, just wanted to raise the question.

Reply:
I assumed that either single or double quote would be valid in HTML. They both seem to work fine, so that counts for something, yes? As long as we are consistent, I have no preference.

Reply:
Unfortunately "it works" is not a usable metric when it comes to HTML validity. Browsers will try to parse almost anything without complaint. I can't find the relevant HTML specs, so I ran some test input through the W3C validator. It accepts both soft and hard quotes (and even unquoted) without warning, so I guess both are according to spec.

So that degrades it to a style issue. All the documentation and 99.9% of the HTML source code out there always uses soft quotes, and that's why at some point in the future I can see myself normalizing all our HTML to soft quotes, as hard quotes in HTML are somewhat of an eye sore to me.

Reply:
Thanks for the research, and I'm fine with whatever format we choose. 

I exported the last two runs in the new report format, and they are now live on the official site:

https://tlscanary.mozilla.org/runs/2017-11-16-09-04-06/
https://tlscanary.mozilla.org/runs/2017-11-15-12-59-15/

\o/